### PR TITLE
chore: merge release branch into develop once it is cut

### PR DESCRIPTION
**Pull request by floralan**
### What does this PR do?
Bring back release branch into develop as soon as it is cut.

### What issues does this PR fix or reference?
 @W-10943603@

### CODE REVIEW

File: scripts/create-release-branch.js [hunk 1]
```diff
@@ -102,5 +92,21 @@ shell.exec('git add lerna.json');
...
+
+// Generate changelog
+const previousBranchName = changeLogGeneratorUtils.getPreviousReleaseBranch(releaseBranchName);
+const parsedCommits = changeLogGeneratorUtils.parseCommits(changeLogGeneratorUtils.getCommits(releaseBranchName, previousBranchName));
+const groupedMessages = changeLogGeneratorUtils.getMessagesGroupedByPackage(parsedCommits, '');
+const changeLog = changeLogGeneratorUtils.getChangeLogText(releaseBranchName, groupedMessages);
+changeLogGeneratorUtils.writeChangeLog(changeLog);
+
+const commitCommand = `git commit -a -m "chore: generated CHANGELOG for ${releaseBranchName}"`;
+shell.exec(commitCommand);
```
Comment (gbockus-sf) [inline scripts/create-release-branch.js:109]: Do we want to merge into develop before we commit the change log?  I fee like that kinda was the point, but hurts my head a little :)

Comment (gbockus-sf) [reply scripts/create-release-branch.js:109]: I went back and looked at the ticket and it does indeed say to do exactly what you're doing. I do feel like we need a code comment here to call out why we do that b/c if I were to revisit this code in 6 months I'll def be tempted to change the order.

Comment (floralan) [reply scripts/create-release-branch.js:110]: @gbockus-sf  Added some comment here.

Comment (floralan) [reply scripts/create-release-branch.js:110]: @gbockus-sf  Added some comment here.

**Review by gbockus-sf — APPROVED** [review]
cool

**Pull request merged**